handlers/reg.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+const docClient = new AWS.DynamoDB.DocumentClient();
+
+module.exports.create = async (event, ctx, callback) => {
+
+  const timestamp = new Date().getTime();
+  const data = JSON.parse(event.body);
+  const studentID = parseInt(data.studentID, 10);
+
+  const params = {
+      Item: {
+          studentID,
+          eventID: data.eventID,
+          status: data.status,
+          createdAt: timestamp,
+          updatedAt: timestamp
+      },
+      TableName: 'biztechRegistration'
+  };
+
+  await docClient.put(params).promise()
+
+  // Update Event count
+  let number = '';
+  let updateExpression = 'set ';
+  switch(data.status) {
+    case 'R':
```

Comment:
Is status a query parameter in the request body? Could it just be `regNum` instead of `R`?

Reply:
`R`, `C`, etc. are the statuses that are being store in registrationTable, while `regNum`, `checkedNum` etc. is the count being store in events table. If you want we could change both to something more compatible like `registered`, `checkedIn`, `waitlisted`, etc.

Reply:
I think it'd be helpful to use words instead of letters, and maybe `registeredCount` or `registeredNum` and `checkedInCount`